src/setupEnzyme.ts

```diff
@@ -0,0 +1,4 @@
+import { configure } from "enzyme";
```

Comment:
this is used only for testing, it shouldn't be under `src`, makes sense to have it at root level